modules/renter/consts.go

```diff
@@ -32,6 +32,6 @@ var (
 	repairQueueInterval = build.Select(build.Var{
 		Dev:      30 * time.Second,
 		Standard: time.Minute * 15,
-		Testing:  time.Minute * 15,
+		Testing:  10 * time.Second,
```

Comment:
we should be consistent about this -- `10 * time.Second` or `time.Second * 10`? I prefer the former, and it's more prevalent in the codebase, the biggest exception being `host/consts.go`.